Classes/Domain/Event/Form/DispatchFormNotificationFinisher.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * Copyright (C) 2018
+ * Nathan Boiron <nathan.boiron@gmail.com>
+ * Romain Canon <romain.hydrocanon@gmail.com>
+ *
+ * This file is part of the TYPO3 NotiZ project.
+ * It is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License, either
+ * version 3 of the License, or any later version.
+ *
+ * For the full copyright and license information, see:
+ * http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace CuyZ\Notiz\Domain\Event\Form;
+
+use TYPO3\CMS\Extbase\SignalSlot\Dispatcher;
+use TYPO3\CMS\Form\Domain\Finishers\AbstractFinisher;
+
+class DispatchFormNotificationFinisher extends AbstractFinisher
+{
+    const NOTIFICATION_DISPATCH_SIGNAL = 'dispatchNotification';
```

Comment:
You could use the value `DispatchNotification` (with a capitalized first letter) instead.

That way you could use this constant as the signal name instead of a string (in `ListFormNotifications:52`)